mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -37,20 +37,17 @@ def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs:
         self._ca_certs = str(Path(certs_dir) / ca_certs)
         self._max_retries = max_retries
 
-    async def make_request(self, interaction_details: Dict[str, str], message: str) -> httpclient.HTTPResponse:
+    async def make_request(self, url: str, headers: Dict[str, str], message: str) -> httpclient.HTTPResponse:
         """Make a request for the specified interaction, containing the provided message. Raises an exception if a
         non-success HTTP status code is returned by the server.
 
-        :param interaction_details: A dictionary containing details of the interaction this message is for.
+        :param url: A string containing the url to send the request to.
+        :param headers: A dictionary containing details of the interaction this message is for.
```

Comment:
This part of the docstring is wrong, the headers are literally just the HTTP headers